### PR TITLE
account deletion statement in privacy

### PR DESCRIPTION
**Pull request by MauriceHendrix**
added confirm delete template and updated the mention of deleting accounts in the privacy page

### CODE REVIEW

**Comment by jonc125** [comment]
It's probably worth adding to this page a note about what will happen to all their models etc - I'm assuming they get deleted? Do we have a test for that?

**Comment by jonc125** [comment]
Ah, looks like we do have some in `accounts/tests/test_views.py`.

*Not* deleting things the person owns is quite tricky. We'd need to implement a mechanism to transfer ownership first, then give the option to auto-transfer everything when deleting the account.

**Comment by MauriceHendrix** [comment]
> Ah, looks like we do have some in `accounts/tests/test_views.py`.
> 
> _Not_ deleting things the person owns is quite tricky. We'd need to implement a mechanism to transfer ownership first, then give the option to auto-transfer everything when deleting the account.

I just tried it, yes it just gets deleted and actually posentially leeds to an error in stories, but ok that's fixable.

File: weblab/templates/registration/account_confirm_delete.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+{% extends "base.html" %}
+
+{% block title %}Delete account{% endblock title %}
+
+{% block content %}
+
+<form action="" method="post">{% csrf_token %}
+    <p>Are you sure you want to delete your account?</p>
+    <p><strong>This operation cannot be undone. Your fills will be deleted.</strong></p>
```
Comment (jonc125) [inline weblab/templates/registration/account_confirm_delete.html:10]:
```suggestion
    <p><strong>This operation cannot be undone. All entities owned by you (models, datasets, experiment results, etc) will be deleted.</strong></p>
```

**Review by jonc125 — APPROVED** [review]

**Pull request merged**